lib/finch.ex

```diff
@@ -251,9 +251,17 @@ defmodule Finch do
           {:ok, acc} | {:error, Exception.t()}
         when acc: term()
   def stream(%Request{} = req, name, acc, fun, opts \\ []) when is_function(fun, 2) do
-    shp = build_shp(req)
-    {pool, pool_mod} = PoolManager.get_pool(name, shp)
-    pool_mod.request(pool, req, acc, fun, opts)
+    start_meta = %{request: req, name: name}
+
+    Finch.Telemetry.span(:stream, start_meta, fn ->
+      shp = build_shp(req)
+      {pool, pool_mod} = PoolManager.get_pool(name, shp)
+      result = pool_mod.request(pool, req, acc, fun, opts)
+
+      end_meta = Map.put(start_meta, :result, result)
```

Comment:
Does it make sense to call this `:response`?

Reply:
@sneako I came up with a different solution in 17ae19a (emit same event but make the result differ).